src/components/BenefitPlanBeneficiariesSearcher.js

```diff
@@ -33,21 +42,42 @@ function BenefitPlanBeneficiariesSearcher({
   beneficiaryExport,
   errorBeneficiaryExport,
 }) {
+  const mm = useModulesManager();
```

Comment:
can we use better extended naming for this variable like 'modulesManager'?

Reply:
Done.